ci-operator/config/openshift/cluster-samples-operator/openshift-cluster-samples-operator-release-4.8__okd.yaml

```diff
@@ -0,0 +1,99 @@
+base_images:
+  base:
+    name: "4.8"
+    namespace: origin
+    tag: base
+  ocp_builder_rhel-7-golang-1.13:
+    name: builder
+    namespace: ocp
+    tag: rhel-7-golang-1.13
+  ocp_builder_rhel-8-golang-1.15-openshift-4.6:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.15-openshift-4.6
+  ocp_builder_rhel-8-golang-1.15-openshift-4.7:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.15-openshift-4.7
+  ocp_builder_rhel-8-golang-openshift-4.6:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-openshift-4.6
+  openshift_release_golang-1.13:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+images:
+- dockerfile_path: Dockerfile.okd
+  from: base
+  inputs:
+    openshift_release_golang-1.13:
+      as:
+      - registry.svc.ci.openshift.org/openshift/release:golang-1.13
```

Comment:
@vrutkovs I don't think it uses 1.13 for OCP: https://github.com/openshift/release/blob/8b814ee6fd2fe28631f3acb89c9890ea75fafd0a/ci-operator/config/openshift/cluster-samples-operator/openshift-cluster-samples-operator-release-4.8.yaml#L36

Reply:
ah, good catch, updated